### DISCUSSION TITLE
Randomized GIF name...

### DISCUSSION THREAD

**Discussion by julianrinaldi** [moralmunky/Home-Assistant-Mail-And-Packages#605] [Q&A]
I understand the decision to have the GIF file name randomized by default, however, is there a way to turn this off in the code? It would be nice to be able to display this gif elsewhere. Specifically, I would like to display it on my Magic Mirror. There's no way to do this since the name is randomized and there's no way to change the URL on the Magic Mirror from the sensor.mail_image_url.

So again, is there any way to turn off the randomization?

**Comment by firstof9** [comment]
No there isn't, you could however setup a webhook automation in Home Assistant that pushes the value of the `sensor.mail_image_url` to your magic mirror if it has some kind of API you can access.
